src/app.js

```diff
@@ -297,6 +298,18 @@ export const start = async args => {
     screen.spawn('git', ['log', args.join('/'), ...gitLogArguments]);
   });
 
+  branchTable.key('y', function() {
+    const selection = parseSelection(this.items[this.selected].content);
+    const gitBranch = selection[2];
+    copyToClipBoard(gitBranch)
+      .then(() => {
```

Comment:
Can we use `async` and `await` to match the pattern of the other functions here instead?

Reply:
done.